.github/workflows/benchmarks.yml

```diff
@@ -30,12 +30,12 @@ defaults:
 
 jobs:
   build:
-    if: github.repository_owner == 'tardis-sn' &&
-      (github.event_name == 'push' ||
-      (!github.event.pull_request.draft) ||
-      github.event_name == 'workflow_dispatch' ||
-      (github.event_name == 'pull_request_target' &&
-      contains(github.event.pull_request.labels.*.name, 'benchmarks')))
+    # if: github.repository_owner == 'tardis-sn' &&
+    #   (github.event_name == 'push' ||
+    #   (!github.event.pull_request.draft) ||
+    #   github.event_name == 'workflow_dispatch' ||
+    #   (github.event_name == 'pull_request_target' &&
+    #   contains(github.event.pull_request.labels.*.name, 'benchmarks')))
```

Comment:
Are these going to remain commented out?

Reply:
I just changed them for testing the workflow, will undo them now.